kratos/utilities/binbased_fast_point_locator.h

```diff
@@ -131,7 +131,9 @@ class BinBasedFastPointLocator
                          array_1d<double, TDim + 1 > & N,
                          Element::Pointer& pelement,
                          ResultIteratorType result_begin,
-                         const unsigned int MaxNumberOfResults)
+                         const unsigned int MaxNumberOfResults,
+                         const double Tolerance = 1.0e-3
```

Comment:
This tolerance is too high for being the standard one. I would have put a tighter tolerance as default and leave users to relax it to their needs. @jcotela and @ipouplana may have different opinion.